Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -380,21 +380,24 @@ static bool isRendererThreadSafe(WebSampleBufferVideoRendering *renderering)
     resetReadyForMoreSample();
 }
 
-void VideoMediaSampleRenderer::decodedFrameAvailable(RetainPtr<CMSampleBufferRef>&& sample)
+VideoMediaSampleRenderer::DecodedFrameResult VideoMediaSampleRenderer::decodedFrameAvailable(RetainPtr<CMSampleBufferRef>&& sample)
```

Comment:
Why RetainPtr<>&&, it does not seem like you make use of it.

Reply:
we are moving it to the queue. existing code